doc/modules/feature_extraction.rst

```diff
@@ -669,7 +669,7 @@ Finally it is possible to discover the main topics of a corpus by
 relaxing the hard assignment constraint of clustering, for instance by
 using :ref:`NMF`:
```

Comment:
not part of this pr, but this also renders incorrectly: it prints the full name of the doc section. Could you make it simply print NMF?